project-base/tests/App/Acceptance/acceptance/OrderCest.php

```diff
@@ -16,8 +16,17 @@
 class OrderCest
 {
     protected const MINIMUM_FORM_SUBMIT_WAIT_TIME = 10;
-    private const TRANSPORT_CZECH_POST_POSITION = 0;
-    private const PAYMENT_CACHE_ON_DELIVERY = 1;
+    protected const TRANSPORT_CZECH_POST_POSITION = 0;
+    protected const PAYMENT_CACHE_ON_DELIVERY = 1;
+
+    protected const DEFAULT_TRANSPORT_NAME = 'Czech post';
+    protected const DEFAULT_PAYMENT_NAME = 'Cash on delivery';
+    protected const DEFAULT_PRODUCT_NAME = 'Defender 2.0 SPK-480';
+
+    protected const DEFAULT_BILLING_STREET = 'Koksární 10';
+    protected const DEFAULT_BILLING_CITY = 'Ostrava';
+    protected const DEFAULT_BILLING_POSTCODE = '702 00';
+    protected const DEFAULT_PHONE = '123456789';
```

Comment:
is it necessary to have these constants protected?

Reply:
Changed